### PR TITLE
docs: remove method warning

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

**Comment by jirikuncar** [comment]
@lnielsen `docs: remove method warning` sounds a bit misleading as if we were removing method warnings.

**Comment by lnielsen** [comment]
Extra commit added.

File: invenio_files_rest/models.py [hunk 1]
```diff
@@ -377,7 +377,7 @@ def get_tags(self):
         return {t.key: t.value for t in self.tags}
 
     @classmethod
-    def create(cls, location=None, storage_class=None):
+    def create(cls, location=None, storage_class=None, **kwargs):
```
Comment (jirikuncar) [inline invenio_files_rest/models.py]: @lnielsen can you include new arguments in docstring?
```
:param \**kwargs: Keyword arguments are forwarded to the class constructor.
```

**Comment by jirikuncar** [comment]
@lnielsen can you separate the version bump commit to new PR while we are improving this PR?

**Pull request merged**